apps/builder/app/builder/features/sidebar-left/sidebar-left.tsx

```diff
@@ -46,59 +47,49 @@ const useHideActiveTabOnPreview = () => {
 const AiTabTrigger = () => {
   const [clientSettings, setClientSetting] = useClientSettings();
   return (
-    <Tooltip side="right" content="AI">
-      <SidebarTabsTrigger
-        aria-label="AI"
-        value={
-          "anyValueNotInTabName" /* !!! This button does not have active state, use impossible tab value  !!! */
-        }
-        onClick={() => {
-          setClientSetting(
-            "isAiCommandBarVisible",
-            clientSettings.isAiCommandBarVisible === true ? false : true
-          );
-        }}
-      >
-        <AiIcon size={rawTheme.spacing[10]} />
-      </SidebarTabsTrigger>
-    </Tooltip>
+    <SidebarButton
+      label="AI"
+      data-state={clientSettings.isAiCommandBarVisible ? "active" : undefined}
+      onClick={() => {
+        setClientSetting(
+          "isAiCommandBarVisible",
+          clientSettings.isAiCommandBarVisible === true ? false : true
```

Comment:
I know your love for `!`
But really in what universe `clientSettings.isAiCommandBarVisible === true ? false : true` is more readable vs
`!clientSettings.isAiCommandBarVisible` 

Even more when I see this:
` clientSettings.isAiCommandBarVisible === true ` I assume that this is done for real reason and the only real reason to compare boolean with `=== true` if it can be undefined or null or even other type
But it's not, it's just boolean.

Reply:
this one was unnecessary actually